README.md

```diff
@@ -20,10 +20,11 @@ There are two major versions of Azure Service Operator: v1 and v2. Consult the b
 
 > Note: ASO v1 and v2 are two totally independent operators. Each has its own unique set of CRDs and controllers. They can be deployed side by side in the same cluster.
 
-| ASO Version | Lifecycle stage | Development status                | Installation options                                                                                                                                                                                    |
-|-------------|-----------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| v2          | Stable          | Under active development.         | [Helm chart](/v2/charts), [GitHub release 2.x](https://github.com/Azure/azure-service-operator/releases). See [installation](https://azure.github.io/azure-service-operator/#installation) for example. |
-| v1          | Beta            | Bug and security fixes primarily. | [Helm chart](/charts), [OperatorHub](https://operatorhub.io/operator/azure-service-operator) or [GitHub release 1.x](https://github.com/Azure/azure-service-operator/releases)                          |
+| ASO Version | Lifecycle stage | Current Release                                                                     | Development status                | Installation options                                                                                                                                                                                    |
+| ----------- | --------------- | ----------------------------------------------------------------------------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| v2          | Stable          | [v2.1.0](https://github.com/Azure/azure-service-operator/releases/tag/v2.1.0)       | Under active development.         | [Helm chart](/v2/charts), [GitHub release 2.x](https://github.com/Azure/azure-service-operator/releases). See [installation](https://azure.github.io/azure-service-operator/#installation) for example. |
```

Comment:
I'm concerned we're putting specific links here that will become out of date when we forget to update them as part of the release process. If we document the need to update this in the release process then it makes the release process more manual.

I accept the need for _some_ links/updates, but every one we add adds more burden on the releaser.

Can we just link https://github.com/Azure/azure-service-operator/releases here instead for v2.0 maybe? The latest release is always at the top of that page so it's pretty much as good as linking to it. OTOH, we can use https://github.com/Azure/azure-service-operator/releases/latest, which is an alias for the latest release, and avoid mentioning the specific version here so we don't have to come update it every release?

Putting the actual version into our readme seems unnecessary since it's already on the front page of the repo: 
![image](https://github.com/Azure/azure-service-operator/assets/8798850/c4a9a129-05f1-4a25-982b-d9be764cf1de)

Reply:
Removed.